All project code/index.js

```diff
@@ -112,25 +112,32 @@ app.get("/profile", (req, res) => {
     let month;
     let day;
     let year;
+    let week;
+    let currDay;
+    let today = new Date();
+    currDay = today.getDate();
 
-    if(req.query.month && req.query.day && req.query.year) {
+    if(req.query.month && req.query.day && req.query.year && req.query.week) {
         month = parseInt(req.query.month);
         day = parseInt(req.query.day);
         year = parseInt(req.query.year);
+        week = parseInt(req.query.week);
     }
     else {
-        let today = new Date();
         today.setDate(today.getDate() - today.getDay());
         month = today.getMonth() + 1;
         day = today.getDate();
         year = today.getFullYear();
+        week = 0;
```

Comment:
What does "week" represent? Why is it initialized to 0?

Reply:
This is a week offset just used to store where the user is in relation to the current date in reality. This is used for the purposes of highlighting the current day (More efficient and readable than checking the entire date at each iteration of the calendar loop)